cypress.json

```diff
@@ -1,6 +1,5 @@
 {
-  "screenshots": {
-    "screenshotOnRunFailure": false
-  },
-  "projectId": "65oh1u"
+    "testFiles": "**/*.feature",
+    "video": false,
```

Comment:
Why are we setting `video: false` here?  Seems like we want videos when doing recorded CI runs...

Reply:
Same reason as for the screenshots: I think this should be explicitly turned on for CI. During development this won't be needed as we're probably debugging in the browser anyways.

These files will be git-ignored, so on the one hand devs won't necessarily notice these files have been created and on the other hand is the amount of disk space used by videos quite a bit.

Reply:
Ah you're right, totally fair!  I'd used the `--video false` flag for that, but putting it in the config file is definitely better if we can override it explicitly when running in CI.  Nice.

Reply:
Ok.. seems like right now that's not an option: kimmobrunfeldt/concurrently#33
So I'm inclined to remove these options from cypress.json until this 3.5 year old issue has been solved..... Do you agree?

Reply:
https://docs.cypress.io/guides/guides/environment-variables.html#Setting

Looks like an env bar would work too?

Reply:
(Untested)

Reply:
It's a bit more complicated. We'd have to set the config with code, depending on an env var, and I wasn't successful in doing so (I didn't try extensively though).

I think for now it's ok to remove it from the config and do this on a new/another branch/PR

Reply:
This works (with `"video": false` in `cypress.json`)

```
CYPRESS_VIDEO=true cypress run
```

I think you're right, it's better not to create screenshots or videos on a dev machine so let's default those to **false** in `cypress.json` (as you had done) and override them with the env vars in `.travis.yml`

Reply:
(no configuration with code required)

Reply:
I've added this functionality for taking screenshots manually as that didn't work.
Works for videos out of the box.